<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoShoping</class>
 <widget class="QWidget" name="GoShoping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1759</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>1421</width>
     <height>381</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>570</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>添加到购物车</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索名称</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>150</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>商品描述</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>价格升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>价格降序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID降序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>GoShoping</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1069</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
